students/NatalieRodriguez/Lesson08/Circle.py

```diff
@@ -0,0 +1,116 @@
+#Natalie Rodriguez
+# Lesson 08: Circle
+# May 12, 2018
+'''
+Goal:
+The goal is to create a class that represents a simple circle.
+
+A Circle can be defined by either specifying the radius or the diameter, and the user can query the circle for either its radius or diameter.
+
+Other abilities of a Circle instance:
+
+Compute the circle’s area
+Print the circle and get something nice
+Be able to add two circles together
+Be able to compare two circles to see which is bigger
+Be able to compare to see if there are equal
+(follows from above) be able to put them in a list and sort them
+You will use:
+
+properties
+a classmethod
+a define a bunch of “special methods”
+
+General Instructions:
+For each step, write a couple of unit tests that test the new features.
+Run these tests (and they will fail the first time)
+Add the code required for your tests to pass.
+    '''
+
+#imports
+
+import math
+
+
+class Circle:
+
+    def __init__(self, in_radius=1):
+        self._radius = in_radius
```

Comment:
A leading underscore conventionally denotes a 'protected' attribute in Python, and I don't think you want to indicate that here. It's also conventional to use the same name for the input parameter and instance attribute that it sets in the constructor.